cmd/osde2ectl/main.go

```diff
@@ -0,0 +1,55 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	// The root import needs to happen before importing the rest of osde2e, as this is what imports the various assets.
+	_ "github.com/openshift/osde2e"
+
+	"github.com/openshift/osde2e/cmd/osde2ectl/create"
+	"github.com/openshift/osde2e/cmd/osde2ectl/delete"
+	"github.com/openshift/osde2e/cmd/osde2ectl/extend"
+	"github.com/openshift/osde2e/cmd/osde2ectl/get"
+	"github.com/openshift/osde2e/cmd/osde2ectl/list"
+	"github.com/spf13/cobra"
+)
+
+var root = &cobra.Command{
+	Use:           "osde2ectl",
+	Long:          "Command line tool for osde2ectl.",
+	SilenceErrors: true,
+	SilenceUsage:  true,
+	// PersistentPreRun: func(cmd *cobra.Command, argv []string) {
```

Comment:
Delete this, but this looks like something that we may want to look into ourselves.